### PR TITLE
Initial Structure of Machine Learning Notebooks

### PR DESCRIPTION
**Pull request by JakeRadMSFT**

### CODE REVIEW

File: machine-learning/01-Intro to Machine Learning.ipynb [hunk 1]
```diff
@@ -1,402 +1,392 @@
...
+        "\n",
+        "\n",
+        "```\n",
+        "var size = new HouseData() { Size = 2.5F };\n",
+        "var price = predictionEngine.Predict(size);\n",
+        "```\n",
+        "\n",
+        "The above code shows how to consume a model that's already been trained. The end result of training a model is a function you can pass some data `HouseData.Size`  to the model and it will give you back a prediction - `Prediction.Price`. \n",
+        "\n",
+        "The above is a simple example (probably too simple) but models can take in many more values.  For instance - [Value Prediction(Regression) with Taxi Dataset](https://raw.githubusercontent.com/dotnet/csharp-notebooks/fa302c12c7494e5f8a5fdbe5d8283d8ff1fb7009/machine-learning/E2E-Value%20Prediction(Regression)%20with%20Taxi%20Dataset.ipynb) -- \n",
```
Comment (beccamc) [inline machine-learning/01-Intro to Machine Learning.ipynb:24]: something funky happened with this url. Problem is the (regression) in the url?

File: machine-learning/01-Intro to Machine Learning.ipynb [hunk 2]
```diff
@@ -1,402 +1,392 @@
...
+        "is a more complex example that takes in `vendor_id`,  `rate_code`, `passenger_count`, `trip_time_in_secs`, `trip_distance`, `payment_type` and then predicts `fare_amount`.\n",
+        "\n",
+        "### How do you create that function?\n",
+        "Machine Learning creates or trains that model by feeding an algorithm a bunch of data and using statistics to predict the values. More details and example below!"
```
Comment (beccamc) [inline machine-learning/01-Intro to Machine Learning.ipynb:28]: How nit-picky can I be? You use the phrase "a bunch" three times in the next two paragraphs.

File: machine-learning/01-Intro to Machine Learning.ipynb [hunk 3]
```diff
@@ -1,402 +1,392 @@
...
+        "using Microsoft.ML;\n",
+        "using Microsoft.ML.Data;"
+      ],
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "Now we are ready to write the code to achieve the machine learning task we need to do. Always start with creating the [MLContext](https://docs.microsoft.com/dotnet/api/microsoft.ml.mlcontext?ranMID=43674&ranEAID=rl2xnKiLcHs&ranSiteID=rl2xnKiLcHs-LuTsrQLVgyEOYaht34D47g&epi=rl2xnKiLcHs-LuTsrQLVgyEOYaht34D47g&irgwc=1&OCID=AID2200057_aff_7795_1243925&tduid=(ir__2m3q0nl02wkf6gcatnnkkvci0e2xvxwafx3xgf9200)(7795)(1243925)(rl2xnKiLcHs-LuTsrQLVgyEOYaht34D47g)()&irclickid=_2m3q0nl02wkf6gcatnnkkvci0e2xvxwafx3xgf9200&view=ml-dotnet) which is the common context for all ML.NET operations"
```
Comment (beccamc) [inline machine-learning/01-Intro to Machine Learning.ipynb:116]: another weird link

File: machine-learning/01-Intro to Machine Learning.ipynb (hunk 2 above)
Comment (JakeRadMSFT) [reply machine-learning/01-Intro to Machine Learning.ipynb:28]: Jeesh I really used it a bunch.

File: machine-learning/01-Intro to Machine Learning.ipynb (hunk 3 above)
Comment (JakeRadMSFT) [reply machine-learning/01-Intro to Machine Learning.ipynb:116]: I'll fix these with ntbk.io short urls once we've settled on content a bit :).

**Review by luisquintanilla — APPROVED** [review]

**Pull request merged**